### ISSUE TITLE
V.1 Create view study set (parent) page

### ISSUE THREAD

**Issue by ashkjones** [PonderTeam/ponder-app#6]
## User Story
As a user, I want to be able to see the title, description, and contents of a study set all in one place, so I know what I will be studying. 

## Sub-tasks
- [ ] Fetch relevant information from the database (if not already cached locally)
- [x] Add title and description of study set
- [ ] Add this set a a recent set for the user in DB
- [ ] Implement description hiding on hold
- [x] Add tabs and container for the flashcard/sequence content 

## Notes
- Depends on #14
